actions/versions.go

```diff
@@ -28,6 +28,10 @@ import (
 type Versions map[string]string
 
 func (v Versions) GetLatest(inputs Inputs) Outputs {
+	return v.GetLatestWithRequest(inputs, nil)
+}
+
+func (v Versions) GetLatestWithRequest(inputs Inputs, f RequestModifierFunc) Outputs {
```

Comment:
If we made `GetLatest` and `SHA256FromURI` variadic along the lines of:
```
func (v Versions) GetLatest(inputs Inputs, mods ...RequestModifierFunc) Outputs {
```
You can use the same function for requests with and without modifiers. It would also be convenient if we need multiples modifiers in the future.